package.json

```diff
@@ -45,75 +44,85 @@
     },
     "lint-staged": {
         "*.{js,ts,html,md,less,json}": [
+            "npm run lint -- --fix",
             "prettier --write",
             "git add"
-        ],
-        "*.ts": "tslint"
+        ]
     },
     "browserslist": [
         "extends @taiga-ui/browserslist-config"
     ],
     "prettier": "@taiga-ui/prettier-config",
     "dependencies": {
-        "@angular/animations": "12.2.17",
-        "@angular/common": "12.2.17",
-        "@angular/compiler": "12.2.17",
-        "@angular/core": "12.2.17",
-        "@angular/forms": "12.2.17",
-        "@angular/platform-browser": "12.2.17",
-        "@angular/platform-browser-dynamic": "12.2.17",
-        "@angular/platform-server": "12.2.17",
-        "@angular/router": "12.2.17",
+        "@angular/animations": "15.2.10",
+        "@angular/common": "15.2.10",
+        "@angular/compiler": "15.2.10",
+        "@angular/core": "15.2.10",
+        "@angular/forms": "15.2.10",
+        "@angular/platform-browser": "15.2.10",
+        "@angular/platform-browser-dynamic": "15.2.10",
+        "@angular/platform-server": "15.2.10",
+        "@angular/router": "15.2.10",
         "@ng-web-apis/common": "3.0.6",
         "@ng-web-apis/universal": "3.0.7",
-        "@nguniversal/express-engine": "12.1.3",
+        "@nguniversal/express-engine": "15.2.1",
         "@tinkoff/ng-dompurify": "4.0.0",
-        "core-js": "2.6.12",
         "dompurify": "3.0.6",
         "express": "4.18.2",
         "ngx-highlightjs": "6.1.3",
-        "rxjs": "6.6.7",
+        "rxjs": "7.8.0",
         "tslib": "2.6.2",
-        "zone.js": "0.11.4"
+        "zone.js": "0.12.0"
     },
     "devDependencies": {
-        "@angular-builders/custom-webpack": "12.1.3",
-        "@angular-devkit/build-angular": "12.2.18",
-        "@angular-devkit/core": "12.2.18",
-        "@angular/cli": "12.2.18",
-        "@angular/compiler-cli": "12.2.17",
-        "@angular/language-service": "12.2.17",
+        "@angular-devkit/build-angular": "15.2.10",
+        "@angular-devkit/core": "15.2.10",
+        "@angular-eslint/builder": "17.1.1",
+        "@angular-eslint/eslint-plugin": "17.1.1",
+        "@angular-eslint/eslint-plugin-template": "17.1.1",
+        "@angular-eslint/schematics": "17.1.1",
+        "@angular-eslint/template-parser": "17.1.1",
+        "@angular/cli": "15.2.10",
+        "@angular/compiler-cli": "15.2.10",
+        "@angular/language-service": "15.2.10",
         "@commitlint/cli": "18.4.3",
         "@commitlint/config-conventional": "18.4.3",
-        "@nguniversal/builders": "12.1.3",
+        "@nguniversal/builders": "15.2.1",
+        "@nrwl/eslint-plugin-nx": "14.1.9",
```

Comment:
why you add `@nrwl/eslint-plugin-nx`?

Reply:
I wanted to add lint target like in maskito, but I changed my mind) the project is too small